test/sanitytest.c

```diff
@@ -49,6 +49,15 @@ static int test_sanity_enum_size(void)
     return 1;
 }
 
+static int test_sanity_int_size(void)
+{
+    if (!TEST_size_t_eq(sizeof(int), 4)
+        || !TEST_int_eq(CHAR_BIT, 8)
+        || !TEST_size_t_lt(sizeof(int), sizeof(int64_t)))
```

Comment:
...  and why `lt` rather than `le`?  Is `int` assume to stay 32 bit for ever and ever?